src/plugins/cordova-plugin-device-motion/sim-host.js

```diff
@@ -197,47 +208,58 @@ function createCanvas() {
     ThreeDee.setLight(-300, -300, 800);
 
     node.addEventListener('mousemove', function (e) {
-        if (_mouseDown && !_shiftKeyDown) {
-            _offsets.x = (_offsets.x + _oldX - e.offsetX) % 360;
-            _offsets.y = (_offsets.y + _oldY - e.offsetY) % 360;
-
-            _alpha = _alpha || 0;
-
-            // enforce gamma in [-90,90] as per w3c spec
-            _gamma = -_offsets.x;
-            if (_gamma < -90) {
-                _gamma = -90;
+        if (_mouseDown) {
```

Comment:
@guillaumejenkins that's right, but you can launch it for any platform you want to support. I always launch it for android, ios, wp8 and windows. Watching the console will not help so much since there's a lot of information being logged. You can wait for the test report.